english/talk_to_neighbour/woman/go-with-scully/ignore-scully/talk-to-alien.md

```diff
@@ -0,0 +1 @@
+You are way more curious about the alien than Scully. You ask the alien its name, it says your name instead.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.